fluidkeys/keyrotate.go

```diff
@@ -209,8 +211,9 @@ func runImportBackIntoGnupg(keys []*pgpkey.PgpKey, passwords map[fingerprint.Fin
 	return
 }
 
-func makeGnupgBackup() error {
-	return fmt.Errorf("not implemented")
+func makeGnupgBackup() (string, error) {
+	filename, err := gpg.BackupHomeDir(fluidkeysDirectory, time.Now())
+	return filename, err
```

Comment:
Yeah I reckon it's better if `makeGnupgBackup` figures out the full dated archive directory then passes that into `gpg.BackupHomeDir`